benchmarking/parameter.py

```diff
@@ -0,0 +1,66 @@
+class Parameter:
+
+    COUNT = 8
+
+    DURATION_SECONDS = 2
+
+    DATASET_FILE_NAME = "MessageContentDataset.csv"
+
+    DIRECTORY = "./generated_files/"
+    
+    HIERARCHY = [
+        "qps", 
+        "prob_of_file",
+        "mean_length_of_text",
+        "std_dev_length_of_text",
+        "mean_length_of_text_with_file",
+        "std_dev_length_of_text_with_file",
+        "mean_file_size",
+        "std_dev_file_size"
+        ]
+
+    EXTREMA = {
+        "qps": {
+            "min": 2, 
+            "max": 4
```

Comment:
Will update these to the final values after completing the entire Content Rich Benchmarking code.